apps/chat/src/store/share/share.epics.ts

```diff
@@ -753,6 +753,92 @@ const discardSharedWithMeFailEpic: AppEpic = (action$) =>
     }),
   );
 
+const deleteSharedConversationFolderEpic: AppEpic = (action$, state$) =>
```

Comment:
Is it possible to combine epics and pass feature type and folders as payload?

Reply:
Not sure that combining here different epics is good idea, because I will need to to do some ugly logic to diff them, but now its simple and easy to understand

Reply:
yes, it looks like it can be combined in one epic with `featureType`

Reply:
@mikitabut , but in current case we need to maintain 4 similar epics which do the same (difference in 1-2 lines)

Reply:
Fixed